tajo-core/src/main/java/org/apache/tajo/master/exec/QueryExecutor.java

```diff
@@ -327,6 +328,11 @@ public void execNonFromQuery(QueryContext queryContext, Session session, String
     LogicalRootNode rootNode = plan.getRootBlock().getRoot();
 
     EvalContext evalContext = new EvalContext();
+
+    //Non From query should be session's time zone. e,g, select to_char(now(), 'yyyy-MM-dd')
+    String timezoneId = queryContext.get(SessionVars.TIMEZONE);
```

Comment:
Non form query should be used SessionVars.TIMEZONE and some builtin functions(now, date_part ..) should be same